dist/connecting_cloud_provider.html

```diff
@@ -321,7 +321,7 @@ <h4 id="choose-an-allocation-strategy">Choose an Allocation Strategy</h4>
               <li>And you select “Ignore non-active status” from “Rules behaviors”</li>
               <li>And you select “Multiply by Size CPU” from “Rules behaviors”</li>
               <li>And you press the “Save” button</li>
-              <li>Then you should see: “The allocation strategy &quot;Provider:4:your-new-cloud Counting:1 Month - Calendar Window Refresh:[<RefreshBehavior: First of the Month>] Rules:[<RulesBehavior: Ignore non-active status>, <RulesBehavior: Multiply by Size CPU>]&quot; was added successfully.”</li>
+              <li>Then you should see: “The allocation strategy &quot;Provider:4:your-new-cloud Counting:1 Month - Calendar Window Refresh:[&lt;RefreshBehavior: First of the Month&gt;] Rules:[&lt;RulesBehavior: Ignore non-active status&gt;, &lt;RulesBehavior: Multiply by Size CPU&gt;]&quot; was added successfully.”</li>
```

Comment:
Maybe because angle brackets mean something special in HTML. Does your output still look OK in a browser?